### PR TITLE
PIP-740 set lab and award with environment variables

### PR DESCRIPTION
**Pull request by p-sud**

### CODE REVIEW

File: accession/__main__.py [hunk 1]
```diff
@@ -38,19 +40,41 @@ def get_parser():
...
+        "or set the DCC_{} environment variable"
+    )
+    lab_prop = "lab"
+    award_prop = "award"
+    if not lab:
+        try:
+            lab = os.environ[f"DCC_{lab_prop.upper()}"]
+        except KeyError as e:
+            raise EnvironmentError(
+                error_msg.format(*2 * [lab_prop], lab_prop.upper())
```
Comment (ottojolanki) [inline accession/__main__.py]:
```suggestion
                error_msg.format(lab_prop, lab_prop, lab_prop.upper())
```

Comment (ottojolanki) [reply accession/__main__.py]: I would spend few more characters to avoid being this clever :)

File: accession/__main__.py [hunk 2]
```diff
@@ -38,19 +40,41 @@ def get_parser():
...
+            ) from e
+    if not award:
+        try:
+            award = os.environ[f"DCC_{award_prop.upper()}"]
+        except KeyError as e:
+            raise EnvironmentError(
+                error_msg.format(*2 * [award_prop], award_prop.upper())
```
Comment (ottojolanki) [inline accession/__main__.py]:
```suggestion
                error_msg.format(award_prop, award_prop, award_prop.upper())
```

Comment (ottojolanki) [reply accession/__main__.py]: Same as above.

File: accession/__main__.py (hunk 1 above)
Comment (p-sud) [reply accession/__main__.py]: A little self-indulgent, I agree. It was already a bad sign when I was looking at that piece of code and it took a couple of seconds to grok what I wrote. Fixed now

File: accession/__main__.py (hunk 2 above)
Comment (p-sud) [reply accession/__main__.py]: fixed

**Pull request merged**